Documentation/configure-kubectl.md

```diff
@@ -18,9 +25,11 @@ $ chmod +x kubectl
 $ mv kubectl /usr/local/bin/kubectl
 ```
 
-Configure your local Kubernetes client using the following commands:
+## Configure `kubectl`
+
+Configure `kubectl` to connect to the target cluster using the following commands, replacing several values as indicated:
 
-* Replace `${MASTER_HOST}`
+* Replace `${MASTER_HOST}` with the address or name of the controller address or name used in previous steps
 * Replace `${CA_CERT}` with the absolute path to the `ca.pem` created in previous steps
```

Comment:
Is it worth rewording these 3 lines that end in "created in previous steps"? If users come to this doc directly, they may not know what we're referring to here.